src/components/ModelDialog/components/ModalContent/styled.ts

```diff
@@ -0,0 +1,12 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+
+import { themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+export const StyledSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  flex-grow: 1;
```

Comment:
What do these three lines of styling do? If i undo it, nothing changes...

Reply:
The display flex and column is so that children will be displayed in a column. The flex-grow: 1 is that is takes up the most space of the header, the content and the footer in a ModalDialog. Otherwise it will be equally divided. It changes the emailpreview if you remove it.